f8a_report/helpers/report_helper.py

```diff
@@ -90,6 +90,31 @@ def cleanup_db_tables(self):
             # Log the message returned from db cursor
             logger.info('%r' % self.cursor.statusmessage)
             logger.info('Cleanup of Worker Result data tables complete')
+
```

Comment:
Can you please add all the tables mentioned in this document. The retention period is also mentioned.

https://docs.google.com/document/d/1dgM0kZ8_mhKy5kBqhApdb95dLv7HmL8ea2VzQQU8FK8/edit

Reply:
Yes.I will update the pr.